src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -860,21 +861,19 @@ public static function get_resolved_theme_uris( $theme_json ) {
 		}
 
 		$theme_json_data = $theme_json->get_raw_data();
-
-		// Top level styles.
-		$background_image_url = isset( $theme_json_data['styles']['background']['backgroundImage']['url'] ) ? $theme_json_data['styles']['background']['backgroundImage']['url'] : null;
-
 		/*
 		 * The same file convention when registering web fonts.
 		 * See: WP_Font_Face_Resolver::to_theme_file_uri.
 		 */
 		$placeholder = 'file:./';
+
+		// Top level styles.
+		$background_image_url = $theme_json_data['styles']['background']['backgroundImage']['url'] ?? null;
 		if (
 			isset( $background_image_url ) &&
 			is_string( $background_image_url ) &&
 			// Skip if the src doesn't start with the placeholder, as there's nothing to replace.
-			str_starts_with( $background_image_url, $placeholder )
-		) {
+			str_starts_with( $background_image_url, $placeholder ) ) {
```

Comment:
Nit: Do we need this whitespace change here?
```suggestion
			str_starts_with( $background_image_url, $placeholder )
		) {
```

Reply:
Hmm, interesting. My PHP code sniffer is set to WordPress Core and it's doing a bit of auto-formatting for me.